tedana/reporting/static_figures.py

```diff
@@ -184,17 +181,18 @@ def carpet_plot(
 
 def comp_figures(ts, mask, comptable, mmix, io_generator, png_cmap):
     """
-    Creates static figures that highlight certain aspects of tedana processing
+    Create static figures that highlight certain aspects of tedana processing.
+
     This includes a figure for each component showing the component time course,
-    the spatial weight map and a fast Fourier transform of the time course
+    the spatial weight map and a fast Fourier transform of the time course.
 
     Parameters
     ----------
     ts : (S x T) array_like
         Time series from which to derive ICA betas
     mask : (S,) array_like
         Boolean mask array
-    comptable : (C x X) :obj:`pandas.DataFrame`
+    comptable : (C x x) :obj:`pandas.DataFrame`
```

Comment:
```suggestion
    comptable : (C x M) :obj:`pandas.DataFrame`
```
(The two lowercase `x`'s looks weird and the second dimension is metrics, so it might as well be an `M`

Reply:
This is the only really minor remaining suggestion I have. I'm going to approve the PR, but if you agree with this suggestion, then make this change before merging